src/it/scala/io/iohk/ethereum/txExecTest/ForksTest.scala

```diff
@@ -63,13 +57,17 @@ class ForksTest extends AnyFlatSpec with Matchers {
     val startBlock = 1
     val endBlock = 11
 
-    protected val testBlockchainStorages = FixtureProvider.prepareStorages(startBlock, fixtures)
+    protected val testBlockchainStorages = FixtureProvider.prepareStorages(endBlock, fixtures)
```

Comment:
Because the `prepareStorages` mocks the storages from the given block down to block zero. So instead of doing it 11 times, inside the foreach, I changed it to be done once for all blocks